code/game/objects/machinery/doors/windowdoor.dm

```diff
@@ -237,6 +279,14 @@
 		return try_to_activate_door(user)
 
 
+/obj/machinery/door/window/do_animate(animation)
```

Comment:
these do_animate procs arent needed, just keep the flick, far as i can tell you arent using them on a timer.

Reply:
/tg/ uses this.